spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'order'
+require 'menu'
+require 'takeaway'
+
+describe 'User Stories' do
+
+  describe 'Menu' do
+    # As a customer
+    # So that I can check if I want to order something
+    # I would like to see a menu§ of dishes with prices
```

Comment:
Style/AsciiComments: Use only ascii symbols in comments. (https://github.com/bbatsov/ruby-style-guide#english-comments)